vivisect/symboliks/common.py

```diff
@@ -303,7 +303,7 @@ def walkTree(self, cb, ctx=None, once=True):
             if idx < len(cur.kids):
                 # sys.stdout.write('+')
                 kid = cur.kids[idx]
-                if once and kid in done:
+                if once and kid._sym_id in done:
```

Comment:
Fixes an issue where if we had two function calls in a symbolik set, and then tried to wipe the AST of architecture specifics, this had a fun tendency to fail spectacularly.